_data/sites.json

```diff
@@ -1249,6 +1249,17 @@
         ]
     },
 
+    {
+        "name": "Codenvy",
+        "url": "https://codenvy.io/dashboard/#/account",
+        "difficulty": "easy",
+        "notes": "Deletion option can be found in the \"Security\" tab",
```

Comment:
Can you clarify what the options are and if it's just a matter of selecting and click delete? This will help us know if something else is needed in the text

Reply:
Options in the "Security" tab are "New password" and "Delete account", both equally visible. You only have to click on the buttons "Delete this account" and "Delete" in the confirmation dialog to get the current account deleted immediately.

Reply:
Alright, thanks!